test/Dialect/Seq/canonicalization.mlir

```diff
@@ -42,7 +42,7 @@ hw.module @FirRegReset(in %clk : !seq.clock, in %in : i32, in %r : i1, in %v : i
   // value.
```

Comment:
Please update the comment / test.

Just curious, does this have an impact on our designs in practice?

Reply:
There was no change in large design 🤔